### DISCUSSION TITLE
mob app

### DISCUSSION THREAD

**Discussion by ghost** [webtorrent/webtorrent#2112] [General]
Do you have a mob app?

**Comment by alxhotel** [comment]
Hi @galopeynet,

I guess you mean mobile app. There isn't a webtorrent app for mobile devices yet

**Comment by ghost** [comment]
I mean mobile app, .No MOBILE APP ☹😥 Poslano iz programa Pošta za Windows 10 Pošiljatelj: AlexPoslano:24. lipnja 2021. 11:30Primatelj: webtorrent/webtorrentKopija: galopeynet; MentionPredmet: Re: [webtorrent/webtorrent] mob app (#2112) Hi @galopeynet,I guess you mean mobile app. There isn't a webtorrent app for mobile devices yet—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
